openedx/core/djangoapps/cors_csrf/tests/test_middleware.py

```diff
@@ -263,15 +264,15 @@ def _assert_cookie_sent(self, response, is_set):
         """Check that the cross-domain CSRF cookie was sent. """
         if is_set:
             self.assertIn(self.COOKIE_NAME, response.cookies)
-            cookie_header = str(response.cookies[self.COOKIE_NAME])
+            cookie_header = six.text_type(response.cookies[self.COOKIE_NAME])
 
-            expected = b'Set-Cookie: {name}={value}; Domain={domain};'.format(
+            expected = 'Set-Cookie: {name}={value}; Domain={domain};'.format(
                 name=self.COOKIE_NAME,
                 value=self.COOKIE_VALUE,
                 domain=self.COOKIE_DOMAIN
             )
             self.assertIn(expected, cookie_header)
-            self.assertIn('Max-Age=31449600; Path=/; secure', cookie_header)
+            self.assertIn('Max-Age=31449600; Path=/; secure'.lower(), cookie_header.lower())
```

Comment:
Add a comment for why we have the `.lower()` on these so that we know why we're doing that.